tests/test_treestatus.py

```diff
@@ -97,16 +97,18 @@ def test_update_form_validate_reason(app):
     form = TreeStatusUpdateTreesForm(
         status="open",
     )
-    assert (
-        form.validate_reason(form.reason) is None
-    ), "No validation required for `open` status."
+    try:
+        form.validate_reason(form.reason)
+    except ValidationError as exc:
+        assert False, f"No validation required for `open` status: {exc}."
```

Comment:
nit: Might be cleaner to just do
```suggestion
        raise AssertionError(f"No validation required for `open` status: {exc}.")
```